network-proto/node.proto

```diff
@@ -26,6 +26,11 @@ message TransactionIds {
     repeated bytes id = 1;
 }
 
+// Identifier of the single transaction.
+message TransactionId {
+    bytes id = 1;
```

Comment:
This reminds me: in `NodeId` the field is named `content`, which is inconsistent with the other ID messages.